python/fledge/plugins/south/person_detection/videostream.py

```diff
@@ -1,47 +1,45 @@
+# FLEDGE_BEGIN
+# See: http://fledge.readthedocs.io/
+# FLEDGE_END
+
+""" Helper module for starting a separate thread for reading frame from the Camera Device
+"""
 
-import cv2
 import logging
+import subprocess
 from threading import Thread
+
+import cv2
+
 from fledge.common import logger
-import subprocess
 
 
-def detectCoralDevBoard():
+_LOGGER = logger.setup(__name__, level=logging.INFO)
```

Comment:
May be keep default level as WARNING.

Reply:
agree. Though, let's keep INFO. We need to know some (even more) information until we go to a stable point.